modules/test/playwright/helpers/HeadlessCommerceAdminCatalogApiHelper.ts

```diff
@@ -0,0 +1,116 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {ApiHelpers} from './ApiHelpers';
+
+export class HeadlessCommerceAdminCatalogApiHelper {
+	readonly apiHelpers: ApiHelpers;
+	readonly basePath: string;
+
+	constructor(apiHelpers: ApiHelpers) {
+		this.apiHelpers = apiHelpers;
+		this.basePath = 'headless-commerce-admin-catalog/v1.0/';
+	}
+
+	async deleteCatalog(
+		apiHelpers = this.apiHelpers,
+		basePath = this.basePath,
```

Comment:
this doesn't look right

Reply:
why do you need to pass again these two parameters?

Reply:
You are correct. 
This was a necessary remnant from when the code was javascript and also utilizing Alessio's old commits. Removed as requested and fixed to correct type declarations.